docs/sources/dashboards/manage-dashboards/index.md

```diff
@@ -136,12 +136,25 @@ A template variable of the type `Constant` will automatically be hidden in the d
 
 The import process enables you to change the name of the dashboard, pick the data source you want the dashboard to use, and specify any metric prefixes (if the dashboard uses any).
 
-### Discover dashboards on Grafana.com
+### Discover dashboards on grafana.com
 
 Find dashboards for common server applications at [Grafana.com/dashboards](https://grafana.com/dashboards).
 
 {{< figure src="/media/docs/grafana/dashboards/screenshot-gcom-dashboards.png" >}}
 
+## Set up generative AI features for dashboards
+
+{{< docs/public-preview product="dashgpt" featureFlag="dashgpt" >}}
```

Comment:
@nmarrs & @SandersAaronD - The way this shortcode renders in the text, it'll say "dashgpt is currently in public preview". Is there are something else you want me to call the feature in this context? Maybe "AI-powered features" per the feature toggle description?

Reply:
The industry seems to have settled on "generative AI" as the buzzterm for those in the know (and also, we use it a good amount), "DashGPT contains Generative AI features for dashboard workflows" might work?

Reply:
This is standardized text where I just plug in a feature name and a feature toggle, so I can't get too fancy with it. What about:

> Generative AI in dashboards is currently in [public preview](https://grafana.com/docs/release-life-cycle/). Grafana Labs offers limited support, and breaking changes might occur prior to the feature being made generally available. Enable the dashgpt feature toggle in Grafana to use this feature. Contact Grafana Support to enable this feature in Grafana Cloud.

Reply:
Sounds good to me.